404 page

AlissonRoss opened the pull request:
Super barebones/simple 404 page for now

nlawler1737 reviewed (approved):
looks good!

lclemente225 reviewed (dismissed):
Looks good

nlawler1737 reviewed (changes requested):
The path to broken.png should be "../../assets/broken.png".

AlissonRoss commented:
Updated the 404 Page to the changes in main and added it to the sidebar and router. 

iPhone 11 Pro 
![image](https://github.com/Sillor/timewise-client/assets/25699172/e08e88e0-83d5-4b0f-beb0-241a3cfac24e)
Galaxy Note 20
![image](https://github.com/Sillor/timewise-client/assets/25699172/b932eaf0-a0d2-45b1-a663-6ab890a891e7)
Desktop:
![image](https://github.com/Sillor/timewise-client/assets/25699172/4ebb16ee-e52c-4e1a-bd32-fb10b5fa6732)

AlissonRoss commented:
> The path to broken.png should be "../../assets/broken.png".

This should be fixed now

nlawler1737 commented on src/timetracker/pages/404-page/404-page.jsx:
```diff
@@ -0,0 +1,12 @@
+export default function PageNotFound() {
+    return (
+        <div className="min-h-screen body text-white">
+            <div className="flex flex-col items-center">
```
I would also add text-center so it doesn't default to left aligned on smaller screens

nlawler1737 reviewed (dismissed):
looks good

lclemente225 reviewed (dismissed):
Looks good, I just want to note that we should have a redirect in the config for this page when the page returns a 404 code.

lclemente225 commented on src/timetracker/components/navbar/Sidebar.jsx:
Question: I noticed that the 404 page is listed in the sidebar. Will it be there on deployment?

AlissonRoss replied:
I added it to main.jsx as a route. Hopefully this helps

lclemente225 commented on src/main.jsx:
```diff
@@ -67,6 +69,10 @@ const router = createBrowserRouter([
+          {
+            path: "/*",
+            element: <PageNotFound/>
+          },
```
Question: I don't have experience with Browser Router, but I was wondering if this path of "/*" for the PageNotFound component was fine to use.

lclemente225 reviewed (approved):
Looks good
